events/nickname_events/wc_nickname_events.txt

```diff
@@ -0,0 +1,156 @@
+###################
+# Nickname Events
+###################
+
+namespace = WCNIC
+
+### the Old Murkeye
+WCNIC.0001 = { # Murloc nickname for when the character rule long
```

Comment:
As @TheGroxEmpire said, it doesn't trigger.
You prob need `birthday.txt` from `Warcraft-Guardians-of-Azeroth-2\common\on_action`.

Reply:
Done in 0112eaab